### ISSUE TITLE
Versions 0.16.16 && 0.16.17 may have a regression

### ISSUE THREAD

**Issue by EyePulp** [jspm/jspm-cli#1363]
Howdy - overnight, I had a problem with an bundle command and the only difference has been updating from 0.16.15 to 0.16.17 between the events (as far as I can tell).

When in the same directory with my config.js, I run:
```
$ /var/projects/foo/bin/node/bin/jspm bundle "src/*.js" "dist/all.js" --inject --minify     
Building the bundle tree for src/*.js...
```
and get:
```
err  Error: Unable to calculate canonical name to bundle file:///var/projects/foo/code/foo/static/js/dist/all.js
         at getCanonicalNamePlain (/var/projects/foo/bin/node/lib/node_modules/jspm/node_modules/systemjs-builder/lib/utils.js:191:13)
         at getCanonicalName (/var/projects/foo/bin/node/lib/node_modules/jspm/node_modules/systemjs-builder/lib/utils.js:102:19)
         at /var/projects/foo/bin/node/lib/node_modules/jspm/lib/bundle.js:104:38
```

I dropped back to  0.16.16 with the same error, and then  0.16.15, which worked, with the command unchanged.  Let me know if I can provide more help.

config.js looks like:
```javascript
System.config({
  baseURL: "/static/js",
  defaultJSExtensions: true,
  transpiler: "babel",
  babelOptions: {
    "sourceMap": false,
    "blacklist": [],
    "optional": [
      "runtime",
      "es7.classProperties",
      "es7.objectRestSpread"
    ]
  },
  paths: {
    "github:*": "jspm_packages/github/*",
    "npm:*": "jspm_packages/npm/*"
  },
...
```

**Comment by guybedford** [comment]
Thanks for reporting. Do you have a `baseURL` set in the package.json file? If you run `jspm init` does the package.json file show up as the current directory? (I take it your current directory is /var/projects/foo/code/foo above?)

**Comment by mikeobrien** [comment]
Exact same problem. Rolling back to 15 fixes it.

**Comment by donspaulding** [comment]
Same problem here, pinning to version 0.16.15 fixed it for me as well.

I too have baseURL set in config.js.

**Comment by EyePulp** [comment]
@guybedford -- I don't have baseUrl specified in package.json

**Comment by guybedford** [comment]
Thanks, fixed in 0.16.18.
